src/App.tsx

```diff
@@ -1,14 +1,89 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useCallback,
+  useMemo,
```

Comment:
for my opinion you didn't have to use 'useMemo' in App. You use new array on every reload

Reply:
I re-render `App` when **1)** `Filter` is changed or **2)** the todo is selected. So I want to re-filter my `filteredTodos` in case 1) and not in case 2). So I need `useMemo`.